blog/2024/10-26-jellyfin-release-10.10.0/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Jellyfin 10.10.0
+description: We are pleased to announce the latest stable release of Jellyfin, version 10.10.0!
+authors: joshuaboniface
+slug: jellyfin-release-10.10.0
+tags: [release]
+---
+
+We are pleased to announce the latest stable release of Jellyfin, version 10.10.0!
+
+This major release brings several new features, improvements, and bugfixes to improve your Jellyfin experience. With our faster release cadence between 10.9.0 and 10.10.0, this release should be far less daunting, so please read on for a quick peek at what's new!
+
+You may upgrade your Jellyfin instances at any time now. For those who were running Unstable builds for testing, we thank you immensely, and you may now switch back to the Stable repository and forcibly reinstall/repull the latest version. As always, **ensure you back up your Jellyfin data and configuration directories before upgrading**. With a major release, it's possible you will hit a bug and want to revert, and to do so, you will need to restore from a backup.
+
+Happy watching!
+
+\- Joshua
+
+{/* truncate */}
+
+## Breaking Changes
+
+* There are no known client support issues in the transition from 10.9.0 to 10.10.0 - all your clients should continue to work as-is without any issues or any forced upgrades, though this may change in the future.
+
+* We have deprecated Raspberry Pi V4L2 hardware transcoding support, and are looking to deprecate Raspberry Pi (i.e. ARMv7) in general for releases past 10.10.0. Raspberry Pi has always been a tricky platform for us, as it's popular but extremely poor in performance, and this has caused a lot of people a lot of headaches. With the RPi5 entirely removing a hardware acceleration engine and the aging of the RPi4, we are taking this action now to ensure it's widely known. This also means that 10.10.x will likely be the last major release of Jellyfin to support 32-bit ARM (`armhf` in Debuntu land), which we will likely drop for 10.11.0. If you are running Jellyfin on old ARM hardware, now is the time to start upgrading. See [PR #1148](https://github.com/jellyfin/jellyfin.org/pull/1148).
```

Comment:
I recall someone saying we want to drop armv7 too for similar reasons, but maybe I'm crazy. Can someone really summarize the situation?

Reply:
And then I guess what I'm asking is @felix920506 @gnattu @nyanmisaka what does it actually mean to "drop RPi support" if we're not touching ARM32 in general? What does that actually look like to users?

Reply:
> I recall someone saying we want to drop armv7 too for similar reasons, but maybe I'm crazy. Can someone really summarize the situation?

We are dropping armv7 yes but Raspberry Pi is using armv8 chips since Pi 3, so deprecating V4L2 and stop building for armhf are two different things.

Reply:
I removed the stuff about arm32 deprecation and left it to "RPi" but I'm still not sure (and thus, I'm sure users will not be sure) what this actually means.

Reply:
More info would be greatly appreciated @felix920506 @gnattu

Reply:
I think letting users know that 10.10 will be the last release that providing 32bit arm builds would be better. Users probably are unsure about what "ARMv7" or "armhf" means, but if we say 32bit arm it is much easier to understand. We require 64bit processor on x86 years ago and this change should be understandable to a lot of users.

Reply:
OK, so the idea *is* to drop armv7 builds, I'm OK with that.

Reply:
I don't think there's a clean way to separate the two paragraphs, but I wrote it like so:

>* We have deprecated Raspberry Pi V4L2 hardware transcoding support, and are looking to deprecate 32-bit ARM support (i.e. `armv7` or `armhf`) in general for releases past 10.10.0. Older, slow ARM systems have always been tricky platforms for us: they're popular but extremely poor in performance, and this has caused a lot of people a lot of headaches. With the RPi5 entirely removing a hardware acceleration engine, and most ARM single-board computers now supporting ARM64, we are taking this action now to ensure it's widely known.  If you are running Jellyfin on old ARM hardware, now is the time to start [looking at an upgrade](https://jellyfin.org/docs/general/administration/hardware-selection). See [PR #1148](https://github.com/jellyfin/jellyfin.org/pull/1148).

Reply:
Tweaked that slightly to make it clear, we're deprecating it for 10.10.0 and looking to remove 32-bit ARM in 10.11.0. I think that's fair to give a 6-month heads up.

Reply:
cc @geerlingguy as out of everyone in the community you might want to be aware ;-)